modules/sdk-coin-ada/src/lib/utils.ts

```diff
@@ -4,6 +4,9 @@ import { KeyPair } from './keyPair';
 
 export class Utils implements BaseUtils {
   validateBlake2b(hash: string): boolean {
+    if (!hash) {
```

Comment:
should this be handled on the platform side? seems a bit counterintuitive that hash is `string` instead of `hash?: string` type but can still possibly be undefined?

Reply:
yes we will going to handle from platform too, but this functioned called from multiple place. We observed a issue where are getting this as undefined value.